.eslintignore

```diff
@@ -1,4 +1,7 @@
 node_modules/
 .next/
 **/generated/*.ts
-theme.ts
\ No newline at end of file
+theme.ts
+packages/prettier-config/
+packages/eslint-config/
+packages/typescript-config/
```

Comment:
as these files won't be updated much i think we can ignore then for now?
otherwise you'd need to add the eslint & prettier configs to each

Reply:
Can they not cover themselves?

Reply:
my thoughts:
- these files are pretty static so i wonder if justifies the overhead
- would we run into circular dependencies if we do add a package as it's own dependency?

i will look into it

Reply:
customized eslint & lint-staged to also cover the config packages